File: C#/RPC/Hayaa.RPC/Config/ConfigHelper.cs
using Hayaa.ConfigSeed.Standard;
using System;
using System.Collections.Generic;
using System.Text;

namespace Hayaa.RPC.Common.Config
{
    internal class ConfigHelper : ConfigTool<RPCConfig, RPCRootConfig>
    {
        private static ConfigHelper _instance = new ConfigHelper();
        private ConfigHelper() : base(DefineTable.RPCComponentID)
        {

        }

        internal static ConfigHelper Instance { get => _instance; }
    }
}

File: Java/src/main/java/hayaa/rpc/common/config/ConfigHelper.java
package hayaa.rpc.common.config;

import Hayaa.ConfigSeed.ConfigTool;

class ConfigHelper extends ConfigTool<RpcConfig,RPCRootConfig> {
    public static ConfigHelper getInstance() {
        return _instance;
    }

    private static ConfigHelper _instance = new ConfigHelper();
    private ConfigHelper(){
        super(DefineTable.RPCComponentID,RpcConfig.class,RPCRootConfig.class);
    }
}
